pwa/src/services/isHomepage.ts

```diff
@@ -0,0 +1,12 @@
+//This function checks whether you are at the homepage considering a prefix value to unsure that code depending on the homepage works accordingly

```

Comment:
I think adding a note that its the prefix set in `config.js` would be a nice addition.